base/c.jl

```diff
@@ -18,7 +24,8 @@ Equivalent to the native `char` c-type.
 """
 Cchar
 
-if is_windows()
+# The ccall is equivalent to Sys.iswindows(), but that's not defined yet
```

Comment:
Changed my mind and just squashed it into the other commit so that AV gets a chance to run now that a fix for the libgit2-online test has been merged.